### ISSUE TITLE
Clean up copyright

### ISSUE THREAD

**Issue by jslee02** [dartsim/dart#160]
+ Update copyright years
+ Fix copyright missing files

**Comment by jslee02** [comment]
We should update the copyright years before releasing DART 6.0.

Also, there are some files contain no copyright headers.

```
dart/planning/PathShortener.cpp: *No copyright* UNKNOWN
dart/planning/PathShortener.h: *No copyright* UNKNOWN
dart/planning/PathPlanner.h: *No copyright* UNKNOWN
dart/gui/Jitter.cpp: *No copyright* UNKNOWN
dart/lcpsolver/odeconfig.h: *No copyright* UNKNOWN
dart/lcpsolver/fastltsolve.cpp: *No copyright* GENERATED FILE
dart/lcpsolver/fastdot.cpp: *No copyright* GENERATED FILE
dart/lcpsolver/fastldlt.cpp: *No copyright* GENERATED FILE
dart/lcpsolver/fastlsolve.cpp: *No copyright* GENERATED FILE
dart/utils/urdf/DartLoader.h: *No copyright* UNKNOWN
dart/utils/urdf/DartLoader.cpp: *No copyright* UNKNOWN
dart/collision/fcl/tri_tri_intersection_test.h: *No copyright* UNKNOWN
dart/dynamics/AssimpInputResourceAdaptor.cpp: *No copyright* UNKNOWN
```

**Comment by jslee02** [comment]
Resolved by #679
